lib/ansible/galaxy/api.py

```diff
@@ -133,6 +133,15 @@ def wrapped(self, *args, **kwargs):
                                    % (method.__name__, ", ".join(versions), ", ".join(available_versions),
                                       self.name, self.api_server))
 
+            # Warn only when we know we are talking to a collections API
+            if common_versions == {'v2'}:
+                display.deprecated(
+                    'The v2 Ansible Galaxy API is deprecated and no longer supported. '
+                    'Ensure that you have configured the ansible-galaxy CLI to utilize an '
+                    'updated and supported version of Ansible Galaxy.',
+                    version='2.20'
```

Comment:
Wouldn't the removal version be 2.21?

Reply:
I'm in the process of updating the example in our docs, but going forward, this is 4 including the version the deprecation is added in.  So 2.20 is correct here.